src/CompositeException.php

```diff
@@ -22,6 +22,9 @@ public function __construct(
         Throwable ...$rest,
     ) {
         $this->rest = $rest;
+        /**
+         * @psalm-suppress PossiblyInvalidArgument
```

Comment:
Not need suppress. See https://github.com/vimeo/psalm/issues/9723#issuecomment-1528755188

It's valid error. `$first->getCode()` can return string.